oar/core/advisory_mgr.py

```diff
@@ -34,7 +34,8 @@ def get_advisories(self):
                 errata_id=v,
                 impetus=k,
             )
-            ads.append(ad)
+            if ad.errata_state != AD_STATUS_DROPPED_NO_SHIP:
```

Comment:
@ming1013 seems like your code base is not up to date. this change is already submitted in https://github.com/openshift/release-tests/pull/103

Reply:
code rebased.